acnportal/algorithms/sorted_algorithms.py

```diff
@@ -86,6 +114,25 @@ def register_interface(self, interface: Interface) -> None:
         if self.max_rate_estimator is not None:
             self.max_rate_estimator.register_interface(interface)
 
+    def register_max_rate_estimator(self, interface: Interface) -> None:
```

Comment:
TODO @sunash This needs to be implemented, right now it's a register_interface clone